docs/src/main/asciidoc/_elytron/Using_the_Elytron_Subsystem.adoc

```diff
@@ -127,6 +127,27 @@ _jboss.server.config.dir_.
 If your directory is located outside of _jboss.server.config.dir_, then
 you need to change the _path_ and _relative-to_ values appropriately.
 
+It is possible to encrypt the filesystem realm by configuring the `credential-store` and `secret-key` attributes for the realm.
+
+To enable encryption for a `filesystem-realm`, there are a couple of attributes that can be configured.
```

Comment:
Might be good to add a heading called Encryption or something like that to make this content stand out.